[App] Re-wording build config warning in the docs (#15570)
Lightning-AI/lightning: docs/source-app/glossary/build_config/build_config_advanced.rst

    - Images from private registries are currently not supported.

.. note::

>     Base image must have python installed. We'll be using this system python to create a virtual environment using

Author: ```suggestion
    Custom docker images must have python installed. We'll be using this system python to create a virtual environment using
```

>     Custom docker images must have python installed. We'll be using this system python to create a virtual environment using

Author: ```suggestion
    Custom docker images must have python installed. We'll use `virtualenv` from this system python to create a virtual environment
```

>     Custom docker images must have python installed. We'll use `virtualenv` from this system python to create a virtual environment

Author: ```suggestion
    Custom docker images must have python installed. We'll use `virtualenv` from this system python to create a virtual environment.
```

>     `virtualenv`. We'll also be configuring the virtualenv to use the packages installed under system's python so your

Author: ```suggestion
    `virtualenv`. We'll also configure the virtualenv to use the packages installed under system's python so your
```

>     `virtualenv`. We'll also configure the virtualenv to use the packages installed under system's python so your

Author: ```suggestion
    We'll also configure the virtualenv to use the packages installed under system's python so your
```
